tests/integration/java-integration/run_local_build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ueo pipefail
+
+SCRIPT_ROOT="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+cd "$SCRIPT_ROOT"
+
+docker run \
+ -it \
```

Comment:
Fixed

Reply:
> Looks pretty reasonable to me, some nitpicks and questions I'd still just want covered.
> 
> Also we should add to CI as a new workflow I think: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
> 
> I'm not sure when to trigger it - what things potentially impact this and should be tested on a PR? At the least I think trigger on the following:
> 
> ```yaml
> on:
>   workflow_dispatch:
>   push:
>     branches:
>       - master
>     paths:
>       - 'tests/integration/java-integration/**'
>   pull_request:
>     branches:
>       - master
>     paths:
>       - 'tests/integration/java-integration/**'
> ```
> 
> That way we can trigger it manually for a specific branch and it will auto-trigger on any changes to these files.

I would run it on all branches. So if someone introduces an issue on a branch one would find out. How can I get the previous built binary?

```yaml
name: Java Integration Tests

on: [push]

jobs:
  build:
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v3
      - name: Set up JDK 17
        uses: actions/setup-java@v3
        with:
          java-version: '17'
          distribution: 'corretto'
      - name: Grab previously built binary
        grab the binary from the built before and put it /path/to/fluentbit  
      - name: Build with Maven
        working-directory: tests/integration/java-integration
        run: mvn clean test
        env:
            FLUENTBIT_COMMAND: "/path/to/fluentbit -c src/test/resources/fluentbit.conf"
```

So if you can help me out with the "grab the binary step" it would be great.

Maybe lets initially run this for amd64 and arm64 and then let's see if you want to add more archs.